autorest/codegen/models/request_builder_parameter_list.py

```diff
@@ -134,7 +135,10 @@ def _add_json_kwarg(
             "See the template in our example to find the input shape. " +
             json_kwarg.description
         )
-        json_kwarg.schema = JSONSchema(namespace="", yaml_data={})
+        if isinstance(body_method_param.schema, (ObjectSchema, DictionarySchema)):
+            json_kwarg.schema = JSONObjectSchema(namespace="", yaml_data={})
```

Comment:
I don't think we need to do `JSONObjectSchema`, I think what we should do if the schema is not an `ObjectSchema`, `DictionarySchema`, or `ListSchema`, we change to `AnySchema`, otherwise we keep the old schema. Then on `HiddenModelObjectSchema`, we type that as `JSONObject`, bc really what we want to do now is "if there is no generated model, but the param type / response type is an ObjectSchema, we generate JSONObject"

Reply:
Thanks @iscai-msft , this help a lot! I didn't recognize this before.

And have updated to use this strategy.